### ISSUE TITLE
Allow Sponge to startup outside a development environment.

### ISSUE THREAD

**Issue by me4502** [SpongePowered/Sponge#127]
Currently Sponge can not start up outside a development environment. We should push to get this working, to allow developers to provide us with bug reports easier.

Last I saw, this was waiting on @AbrarSyed

**Comment by Mumfrey** [comment]
We know, but abrar is currently away and this is already covered by SpongePowered/Mixin#4 so this issue doesn't really achieve anything.
